src/data/animations/battle-anims.ts

```diff
@@ -231,16 +281,26 @@ export abstract class BattleAnim {
     let r = anim?.frames.length ?? 0;
     let f = 0;
 
+    /**
+     * Frames are set for each asset of the animation
+     * every 3 ms. This includes the user, target, and all graphics.
```

Comment:
```suggestion
     * every 50 ms. This includes the user, target, and all graphics.
```
`getFrameMs(frameCount)` turns a number of frames into a number of milliseconds.
`getFrameMs(3)` = `return Math.floor((1 / 60) * 1000 * 3);` = `50`
![image](https://github.com/user-attachments/assets/02864377-13ce-47a2-9a61-7040616e5b0f)

It would be helpful if there were TSDocs on `getFrameMs()` lol.